### PR TITLE
Feature/m2 8061 changing terms of service

### PR DESCRIPTION
**Pull request by felipeMetaLab**
### 📝 Description

🔗 [Jira Ticket M2-8061](https://mindlogger.atlassian.net/browse/M2-8061)
#### This PR includes the following updates:
* Added Privacy Policy links to the Sign In, Sign Up, and About pages.
* Added Terms of Service link to the About page.
* Improved handling of line breaks for the Greek language.
* Created a helper function to streamline link sharing across components.
* Additionally, some minor fixes were made to enhance overall functionality.

### 📸 Screenshots
signIn:
Screenshot 2024-10-25 at 10 52 25 AM
achments/assets/053aff8f-b4fb-4578-b658-104a34ebc3bf">
Screenshot 2024-10-25 at 11 48 13 AM

signUp:
Screenshot 2024-10-25 at 11 46 44 AM
Screenshot 2024-10-25 at 11 46 56 AM
Screenshot 2024-10-25 at 11 47 08 AM

ENGLISH:
Screenshot 2024-10-23 at 2 55 15 PM

GREEK BEFORE:
Screenshot 2024-10-23 at 2 39 40 PM

GREEK AFTER:
Screenshot 2024-10-23 at 2 55 51 PM
Screenshot 2024-10-23 at 2 56 03 PM

FRENCH:
Screenshot 2024-10-23 at 3 03 02 PM

### 🪤 Peer Testing

* lauch the app
#### to test in english: 
* navigate to signIn, signUp  and check the new privacy policy link ( I left some prints showing where its located )
* after the sigIn go to about screen clicking the button at the bottom of the first screen,  scroll down and check the new pricavy policy and terms of service link.
#### to test in other languages:
* go to user settings and change the language
* now perform the same test mentioned before in about page, signIn and SignUp 

### ✏️ Notes

Any question let me know, thanks.

### CODE REVIEW

File: src/screens/ui/SignUpScreen.tsx [hunk 1]
```diff
@@ -57,33 +57,33 @@ export const SignUpScreen: FC = () => {
...
-                      openUrl('https://mindlogger.org/privacy-policy')
+                      openUrl('https://mindlogger.org/terms-of-service')
                     }
-                    accessibilityLabel="privacy_policy_link"
                   >
-                    {t('auth:privacy_policy')}
+                    {t('auth:terms_of_service')}
+                  </Text>
+                  <Text fontSize={16} color="$white" ta="center">
+                    and
```
Comment (ChaconC) [inline src/screens/ui/SignUpScreen.tsx:74]: this needs to be translated

File: src/screens/ui/SignUpScreen.tsx [hunk 2]
```diff
@@ -57,33 +57,33 @@ export const SignUpScreen: FC = () => {
...
                     textDecorationLine="underline"
-                    accessibilityLabel="terms_of_service_link"
                     onPress={() =>
-                      openUrl('https://mindlogger.org/terms-of-service')
+                      openUrl('https://mindlogger.org/privacy-policy')
                     }
+                    accessibilityLabel="privacy_policy_link"
                   >
-                    {t('auth:terms_of_service')}
+                    {t('auth:privacy_policy')}
```
Comment (ChaconC) [inline src/screens/ui/SignUpScreen.tsx:86]: missing `.` after "Privacy"

Comment (felipeMetaLab) [reply src/screens/ui/SignUpScreen.tsx]: good observation, I will add it.

File: src/screens/ui/SignUpScreen.tsx (hunk 1 above)
Comment (felipeMetaLab) [reply src/screens/ui/SignUpScreen.tsx]: done, thank you.

**Pull request merged**